resources/options_lookup.tsv

```diff
@@ -423,6 +423,7 @@ Occupants	4	ResidentialScheduleGenerator	num_occupants=4
 Occupants	5	ResidentialScheduleGenerator	num_occupants=5																																												
 Occupants	6	ResidentialScheduleGenerator	num_occupants=6																																												
 Vacancy Status	Occupied	ResidentialScheduleGenerator	vacancy_start_date=NA	vacancy_end_date=NA																																											
+Vacancy Status	Seasonal or occasional use	ResidentialScheduleGenerator	vacancy_start_date=January 1	vacancy_end_date=December 31																																											
```

Comment:
@afontani Do you need an additional "vacancy scaling factor" or something similar to move forward with this?

Reply:
@joseph-robertson: I don't think so, I think we are going to take the approach that X% of the units are vacant at any given time. I am going to remove the "Seasonal or occasional use" option. There will only be vacant and occupied options for now until we have more data on vacations.